### ISSUE TITLE
Calculation of Mulliken Electronegativity

### ISSUE THREAD

**Issue by sedaoturak** [lmmentel/mendeleev#115]
Hi!
I realized that during Mulliken electronegativity calculations, electron affinity is not taken into account. When I checked the source code, I think this is related to last if condition. Can you please check if there is an issue, or do I miss something?

mendeleev version: 0.13.1
python version: 3.10
platform: google colab

Below I also shared a code snippet for only one element but I got the same issue for other elements, as well.

## To Reproduce
```
from mendeleev import element

Ni = element('Ni')
print('Mulliken with function:', Ni.electronegativity('mulliken'))
print('Mulliken with manual calculation:', (Ni.ionenergies[1]+Ni.electron_affinity)/2)
print('Half of first ionization energy:', (Ni.ionenergies[1])/2)
```
The output is:
```
Mulliken with function: 3.8199385
Mulliken with manual calculation: 4.3979385
Half of first ionization energy: 3.8199385
```

**Comment by lmmentel** [comment]
Hey @sedaoturak, thanks a lot for reporting this. I verified it on my end and indeed there was is a bug in the `mulliken` function. It seems the extra flags captured all the edge cases but did not cover the main case so this was a pretty severe bug. I dropped the extra flags since I no longer feel they are that useful and added  tests to verify all is OK. Once #116 passes CI I'll try to cut a fresh release with the fix :crossed_fingers:
